packages/firestore/CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Unreleased
 - [changed] Removed eval()-based fallback for JSON parsing, allowing SDK to
   be used in environments that prohibit eval().
+- [feature] Added a garbage collection process to on-disk persistence that
+  removes older documents. This is enabled by default, and the SDK will attempt
+  to periodically clean up older, unused documents once the on-disk cache passes
+  a threshold size (default: 40 MB). This threshold can be configured by
+  the changing the setting `cacheSizeBytes` in `FirestoreSettings`. It must be
```

Comment:
"the changing" => "changing"

Since JS isn't actually typed, consider changing "in `FirestoreSettings` to "in the settings passed to Firestore.settings()"